packages/vuetify/src/components/VCalendar/VCalendar.ts

```diff
@@ -332,7 +332,12 @@ export default CalendarWithEvents.extend({
           }
         }
 
-        categories = categories.filter(v => Object.keys(categoryMap).includes(v.categoryName))
+        categories = categories.filter((v: CalendarCategory) => {
+          if (typeof v === 'object' && v.categoryName) {
+            return Object.keys(categoryMap).includes(v.categoryName)
```

Comment:
`categoryMap.hasOwnProperty(v.categoryName)`?
also please change `v` to something meaningful (`category` for example)

Reply:
why resolved without action?

Reply:
Sorry about that. Apparently I hadn't pushed my latest changes.